### DISCUSSION TITLE
Instructions for using the sample-config-advanced.json

### DISCUSSION THREAD

**Discussion by billydu1980** [Ylianst/MeshCentral#6092] [General]
Dear All:
Can you share a help document for a detailed instructions of the parameters in the sample-config-advanced.json file.

regard.
Billy Duu

**Comment by si458** [comment]
the sample-config-advanced.json is simple all the valid options for your config.json.
the is no documentation sadly as we do need to update it! https://github.com/Ylianst/MeshCentral/tree/master/docs/docs
https://ylianst.github.io/MeshCentral/
but you basically load the meshcentral-config-schema.json into VScode, https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json
then when you open up your config.json it will predict the options and the is descriptions what each thing means :)

**Comment by billydu1980** [comment]
meshcentral-config-schema is helpful, Thanks so much.

Duu
